src/modules/0.0.16/aws_secret/index.ts

```diff
@@ -68,7 +69,7 @@ async function getSecret(client: SecretsManager, secretId: string) {
   return undefined;
 }
 
-const getAllSecrets = paginateBuilder<SecretsManager>(
+const getAllSecrets = paginateBuilder<SecretsManager | SecretsManagerClient>(
```

Comment:
Wait, how would this work? If the `paginateListSecrets` isn't on the `SecretsManager` object, won't it fail when you pass that client to it?

Reply:
it was a leftover sorry. Tests passed because it's not being called with that client